ibm/service/vpc/resource_ibm_is_vpn_server_client.go

```diff
@@ -0,0 +1,179 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func ResourceIBMIsVPNServerClient() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMIsVPNServerClientDisconnect,
```

Comment:
All Create, Read and Update handles same method?

Reply:
Yes,